pkg/collector/corechecks/net/ntp/ntp.go

```diff
@@ -76,6 +76,20 @@ func (c *NTPCheck) String() string {
 
 // for testing
 var getCloudProviderNTPHosts = cloudproviders.GetCloudProviderNTPHosts
+var getLocalDefinedNTPServersFunc = getLocalDefinedNTPServers
+
+// stringSlicesEqual compares two string slices for equality
+func stringSlicesEqual(a, b []string) bool {
```

Comment:
this function also asserts the order of the elements is the same. I think for this check we don't care about the order.